sysbuild/CMakeLists.txt

```diff
@@ -68,6 +68,16 @@ function(${SYSBUILD_CURRENT_MODULE_NAME}_pre_cmake)
   cmake_parse_arguments(PRE_CMAKE "" "" "IMAGES" ${ARGN})
   restore_ncs_vars()
 
+  if(NOT SB_CONFIG_PARTITION_MANAGER)
+    # Disable any Kconfigs for child images in the main application
+    foreach(config NCS_SAMPLE_EMPTY_APP_CORE_CHILD_IMAGE NCS_INCLUDE_RPMSG_CHILD_IMAGE NCS_SAMPLE_REMOTE_SHELL_CHILD_IMAGE
```

Comment:
isn't some of this handled through e35d9e2d94e8bcda399568819a47bc6e70a6fd2a ?

afaik the `BUILD_STRATEGY` settings will not be available with linked sha.

Reply:
Yes, this was an earlier try to get them unselected, could trim some of the options

Reply:
Fixed